chunky/src/java/se/llbit/chunky/block/Block.java

```diff
@@ -119,7 +105,7 @@ public boolean isBlockWithEntity() {
     return false;
   }
 
-  public Entity toEntity(Vector3 position) {
+  public Entity[] toEntity(Vector3 position) {
```

Comment:
Why was this change necessary? Are there blocks that require multiple entities? Cake with candles? :thinking:

Reply:
We need to break this PR down into smaller changes that can actually be merged, that would probably be simpler than trying to resolve the conflicts.

I'd merge #1376 so this change could be dropped from this PR.

Reply:
Okay; I have reverted the changes to entities in favor of #1376.